Nickvision.Aura/Keyring/PasswordGenerator.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using System.Collections.Generic;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Nickvision.Aura.Keyring;
+
+[Flags]
+public enum PasswordContent
+{
+    Numeric = 1,
+    Uppercase = 2,
+    Lowercase = 4,
+    Special = 8
+}
+
+/// <summary>
+/// Random password generator
+/// </summary>
+public class PasswordGenerator
+{
+    private readonly List<char> _chars;
+    private PasswordContent _contentFlags;
+    
+    /// <summary>
+    /// Construct password generator
```

Comment:
`Constructs a PasswordGenerator`

Reply:
just this or should I change every comment?

Reply:
I mean, I generally prefer to use infinitive instead of 3rd form in comments, and until now this wasn't an issue, but if it is, I will use 3rd form everywhere

Reply:
If you have any other "construct" comments yes please make them in the format `Constructs a(n) ClassName`

Reply:
I mean other comments, like "Generate new password"

Reply:
Ah yes the action should be third-person so `Generates` instead of `Generate`

Reply:
*looking at MPVSharp comments with sorrow...* 😅

Reply:
Sorry I never noticed before!! :)